xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexAdapters.java

```diff
@@ -245,7 +245,9 @@ public static Balance adaptBalance(BittrexBalance balance) {
     List<UserTrade> trades = new ArrayList<>();
 
     for (BittrexUserTrade bittrexTrade : bittrexUserTrades) {
-      trades.add(adaptUserTrade(bittrexTrade));
+      if (!isOrderWithoutTrade(bittrexTrade)) {
```

Comment:
by writing a test with the example JSON from Bittrexes documentation I found out that our implementation has a bug and generates a trade in getTransactionHistory with price = 0 if there was an order which didn't buy / sell anything